draft-mouris-cfrg-mastic.md

```diff
@@ -1130,14 +1130,23 @@ Each output share consists of the truncated payload for each VIDPF prefix,
 flattened into a single vector. Aggregation involves simply adding these up:
 
 ~~~ python
-def aggregate(self,
-              agg_param: MasticAggParam,
-              out_shares: list[list[F]],
-              ) -> list[F]:
-    agg_share = self.empty_agg(agg_param)
-    for out_share in out_shares:
-        agg_share = vec_add(agg_share, out_share)
-    return agg_share
+def agg_init(self, agg_param: MasticAggParam) -> list[F]:
+    return self.empty_agg(agg_param)
+
+def agg_update(self,
+                agg_param: MasticAggParam,
+                agg_share: list[F],
+                out_share: list[F]) -> list[F]:
+    return vec_add(agg_share, out_share)
```

Comment:
No need to cast?

Reply:
cast what into what?

Reply:
the return value to list[F], as in merge

Reply:
no need